protocols/yungapedao/index.json

```diff
@@ -0,0 +1,8 @@
+{
+  "cname": "YungApeDao",
```

Comment:
please change all the letters to small , this is an parameter to retrieve protocols and will not displayed.